src/pages/main/footer/Footer.js

```diff
@@ -0,0 +1,157 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { GiTreeBranch } from 'react-icons/gi';
+import { IoLogoApple } from 'react-icons/io5';
+
+const Footer = () => {
+  useEffect(() => {
+    fetch('data/Footer.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setFooterData(data);
+      });
+  }, []);
+
+  const [footerData, setFooterData] = useState([]);
+  return (
+    <FooterWrapper>
+      <FooterInner>
+        <WrapInfo>
+          <LogoDiv>
+            <FooterLogo />
+            <LogoQuote>
+              Talk is cheap. <br />
+              Show me the code.
+            </LogoQuote>
+            <QuoteAuthor>Linus Torvalds</QuoteAuthor>
+          </LogoDiv>
+          <FooterUl>
+            {footerData.map(({ id, name }) => {
+              return (
+                <FooterLi key={id}>
+                  <FooterA href="#!">{name}</FooterA>
+                </FooterLi>
+              );
+            })}
```

Comment:
- 아래와 같이 하면 될 것 같습니다.
```suggestion
            {footerData.map(({ id, name }) => (
              <FooterLi key={id}>
                <FooterA href="#!">{name}</FooterA>
              </FooterLi>
            ))}
```